resources/assets/lib/notification-widget/item.tsx

```diff
@@ -8,8 +8,7 @@ import { NotificationContext } from 'notifications-context';
 import NotificationDeleteButton from 'notifications/notification-delete-button';
```

Comment:
timestamp is missing hover tooltip

Reply:
side note, I think the timestamp hover on user cards is also missing 👀